DependencyInjection/AdmingeneratorGeneratorExtension.php

```diff
@@ -269,13 +269,15 @@ private function registerGeneratedFormsAsServices(ContainerBuilder $container)
     private function registerFormsServicesFromGenerator(array $generatorParameters, array $builders, ContainerBuilder $container)
     {
         $modelParts = explode('\\', $generatorParameters['model']);
-        $model = array_pop($modelParts);
+        $model = strtolower(array_pop($modelParts)); // @TODO: BC Support, remove it starting from v.3.0.0
```

Comment:
Minor optimisation, no need to call 3 times "strtolower". Comment is added for future release when BC is ok.